sdk/keyvault/azsecrets/models.go

```diff
@@ -14,13 +14,35 @@ import (
 
 // DeletedSecretBundle - A Deleted Secret consisting of its previous id, attributes and its tags, as well as information on when it will be purged.
 type DeletedSecretBundle struct {
```

Comment:
Is there a reason for `DeletedKeyBundle` being renamed to `DeletedKey` but not dropping "Bundle" here?

Reply:
Haven't gotten to secrets yet. I'm going to undo the changes to secrets and do it in another PR.